src/main/java/com/stripe/model/Product.java

```diff
@@ -166,6 +168,18 @@ public void setURL(String url) {
 		this.url = url;
 	}
 
+	public String getStatementDescriptor() {
+		return statementDescriptor;
+	}
+
+	public void setStatementDescriptor(String statementDescriptor) {
+		this.statementDescriptor = statementDescriptor;
+	}
+
+	public String getType() {
+		return type;
+	}
+
```

Comment:
You're missing a setter for `type` (not a big deal since setters are only used in tests, but we have setters ~everywhere so might as well be consistent).